### PR TITLE
[ebay.co.uk] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/issues/13603

### CODE REVIEW

**Comment by youdly** [comment]
`postorder` redirects to `signin` for me, which is functional. I'll see whether travis throws the same error again.

**Comment by J0WI** [comment]
@youdly fetch tests are still throwing some errors.

**Comment by J0WI** [comment]
Travis exposed another broken host.

File: src/chrome/content/rules/ebay.com.xml [hunk 1]
```diff
@@ -36,8 +39,10 @@
 		- hub.motors	(m)
 		- my	(r)
 		- myworld	(m)
+		- partnernetwork	(t)
 		- rtm	(m)
 		- shop	(m)
+		- securethumbs	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.com.xml]: WFM again

File: src/chrome/content/rules/ebay.com.xml [hunk 2]
```diff
@@ -36,8 +39,10 @@
 		- hub.motors	(m)
 		- my	(r)
 		- myworld	(m)
+		- partnernetwork	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.com.xml]: WFM again

File: src/chrome/content/rules/ebay.co.uk.xml [hunk 3]
```diff
@@ -65,9 +65,11 @@
 		- my	(r)
 		- myworld	(r)
 		- neighbourhood	(m)
+		- notifyweb	(SSL error)
 		- o2imode	(m)
 		- o2wap	(m)
 		- orange	(m)
+		- pages		(h)
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.co.uk.xml]: WFM (again?)

File: src/chrome/content/rules/ebay.co.uk.xml [hunk 4]
```diff
@@ -253,14 +258,12 @@
 	<target host="mesg.ebay.co.uk" />
 	<target host="www.mip.ebay.co.uk" />
 	<target host="www.mipapplication.ebay.co.uk" />
-	<target host="notifyweb.ebay.co.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/ebay.co.uk.xml]: WFM

**Comment by J0WI** [comment]
Please don't force push if you update this branch again.

File: src/chrome/content/rules/ebay.co.uk.xml (hunk 4 above)
Comment (youdly) [reply src/chrome/content/rules/ebay.co.uk.xml]: can't resolve host

**Comment by youdly** [comment]
Restored hosts

**Pull request merged**